frontends/infinite-corridor/src/pages/user-lists/ManageListDialogs.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useCallback, useState } from "react"
+import UpsertListDialog from "./UpsertListDialog"
+import BasicDialog from "../../components/BasicDialog"
+import { useToggle } from "ol-util"
+import type { UserList } from "ol-search-ui"
+import { useDeleteUserList } from "../../api/learning-resources"
+
+type CreateListDialogProps = {
+  open: boolean
+  onClose: () => void
+}
+const CreateListDialog: React.FC<CreateListDialogProps> = ({
+  open,
+  onClose
+}) => {
+  return <UpsertListDialog open={open} onClose={onClose} title="Create list" />
+}
+
+type EditListDialogProps = {
+  onClose: () => void
+  resource: UserList | null
+}
+const EditListDialog: React.FC<EditListDialogProps> = ({
+  onClose,
+  resource
+}) => {
+  return (
+    <UpsertListDialog
+      open={!!resource}
+      resource={resource}
+      onClose={onClose}
+      title="Edit list"
+    />
+  )
+}
+
+type DeletionDialogProps = {
+  onClose: () => void
+  resource: UserList | null
+}
+const DeletionDialog: React.FC<DeletionDialogProps> = ({
```

Comment:
Nit: should this and the props be `DeleteListDialog` to align to the other two dialogs' naming convention?